llama.h

```diff
@@ -11,6 +11,7 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <stdbool.h>
+#include <string>
```

Comment:
Should not add C++ headers in the C-style API

Reply:
Also a forgotten artefact, will remove.

Reply:
Done.